packages/contracts/contracts/registry/PluginRepoRegistry.sol

```diff
@@ -41,13 +44,17 @@ contract PluginRepoRegistry is InterfaceBasedRegistry {
     /// @notice Registers a plugin repository with a name and address.
     /// @param name The name of the PluginRepo.
     /// @param pluginRepo The address of the PluginRepo contract.
-    function registerPluginRepo(string calldata name, address pluginRepo)
-        external
-        auth(REGISTER_PLUGIN_REPO_PERMISSION_ID)
-    {
+    function registerPluginRepo(
+        string calldata name,
```

Comment:
if `name` is the subdomain, I suggest we take the chance and call it `ensSubdomain`. The DAO name will live on a JSON file

Reply:
what does the DAO name have to do with `registerPluginRepo`

Reply:
Mixing too many things lately, sorry. But still, the same idea as above.

The plugin name will be on the metadata. On-chain, we handle the ENS subdomain. I suggest we also rename `name` to `ensSubdomain`